tests/usb/usb.go

```diff
@@ -95,14 +97,30 @@ var _ = Describe("[Serial][sig-compute][USB] host USB Passthrough", Serial, deco
 			vmi.Spec.Domain.Devices.HostDevices = hostDevs
 			vmi, err = virtClient.VirtualMachineInstance(util.NamespaceTestDefault).Create(context.Background(), vmi)
 			Expect(err).ToNot(HaveOccurred())
-			libwait.WaitForSuccessfulVMIStart(vmi)
+			vmi = libwait.WaitForSuccessfulVMIStart(vmi)
 			Expect(console.LoginToCirros(vmi)).To(Succeed())
 
 			By("Making sure the usb is present inside the VMI")
 			Expect(console.SafeExpectBatch(vmi, []expect.Batcher{
 				&expect.BSnd{S: fmt.Sprintf("%s\n", cmdNumberUSBs)},
 				&expect.BExp{R: console.RetValue(fmt.Sprintf("%d", len(deviceNames)))},
 			}, 15)).To(Succeed(), "Device not found")
+
+			By("Verifying ownership is properly set in the host")
+			domainXml, err := tests.GetRunningVMIDomainSpec(vmi)
+			Expect(err).ToNot(HaveOccurred())
+
+			for _, hostDevice := range domainXml.Devices.HostDevices {
+				if hostDevice.Type != api.HostDeviceUSB {
+					continue
+				}
+				addr := hostDevice.Source.Address
+				path := fmt.Sprintf("%sdev/bus/usb/00%s/00%s", pkgUtil.HostRootMount, addr.Bus, addr.Device)
+				cmd := []string{"stat", "-c", `"%u %g"`, path}
+				stdout, err := tests.ExecuteCommandInVirtHandlerPod(vmi.Status.NodeName, cmd)
+				Expect(err).ToNot(HaveOccurred())
+				Expect(stdout).Should(ContainSubstring("107 107"))
```

Comment:
nit: I would rather be more strict here and check for equality

```suggestion
				Expect(stdout).Should(BeEqual("107 107"))
```